CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+## Coding style
+
+* HOL source code
+  - Spaces, no tabs
+  - No unicode
+  - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+## Branch policy
+
+### tags
+
+tags should have as many **completed features** as possible:
+ - no cheat
+ - must correctly compile
+ - self tests must succeed
+ - code should be tested
+
+Follow these instructions whenever you merge to master:
+  - `grep` for "cheat"
+  - check that the `README` is up to date (especially tool status)
+  - find a reviewer for your Pull Request
+
+### `master` branch
+
+`master` is the branch where every feature is available, but not necessarily finalized:
+  - Can cheat, but has to be avoided
```

Comment:
Contradicts what is said above.

Reply:
that is true. i think we wrote this here together and didn't fix the other one. and this is also the reason why you asked for the other one now. until now we didn't need cheat in there, so let's see how long we can keep it like this.